metastats-infrastructure/src/test/groovy/life.qbic.metastats/ConditionParserSpecification.groovy

```diff
@@ -0,0 +1,48 @@
+package life.qbic.metastats
+
+
+import life.qbic.metastats.filter.ConditionParser
+import life.qbic.xml.manager.StudyXMLParser
+import life.qbic.xml.properties.Property
+import spock.lang.Specification
+
+class ConditionParserSpecification extends Specification {
+
+    String condition = """<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+            <qexperiment>
+            <technology_type name="Transcriptomics"/>
+            <qfactors>
+            <qcategorical label="genotype">
+            <qcatlevel value="wildtype">
+            <entity_id>QFSVIXXXXY</entity_id>
+            </qcatlevel>
+            <qcatlevel value="mutant">
+            <entity_id>QFSVIXXXXX</entity_id>
+            </qcatlevel>
+    </qcategorical>
+    </qfactors>
+    </qexperiment>
+    """
+
+
+    def "xml parsing retrieves the right number of Conditions"() {
```

Comment:
Nit:
```suggestion
    def "xml parsing retrieves the correct number of Conditions"() {
```

Reply:
I cannot see where the number of conditions is tested here.

Reply:
You are right, I renamed the test